intel_extension_for_transformers/neural_chat/examples/finetuning/multi_modal/eval/mmmu_eval/run_llava.py

```diff
@@ -17,7 +17,7 @@
 from optimum.habana.transformers.generation.utils import MODELS_OPTIMIZED_WITH_STATIC_SHAPES
 if "llava" not in MODELS_OPTIMIZED_WITH_STATIC_SHAPES:
     MODELS_OPTIMIZED_WITH_STATIC_SHAPES.append("llava")
-from optimum.habana.transformers.modeling_utils import adapt_transformers_to_gaudi
+from intel_extension_for_transformers.transformers.modeling.modeling_gaudi import adapt_transformers_to_gaudi
```

Comment:
I am not sure whether the chatbot also import models from internal not from habana_optimum @lvliang-intel